File: Common/InMotionGIT.Common/Services/Contracts/StoredProcedureResult.vb
Imports System.Runtime.Serialization

Namespace Services.Contracts

    <DataContract()>
    Public Class StoredProcedureResult

        <DataMember()>
        Public Property RowAffected As Integer

        <DataMember()>
        Public Property OutParameter As New Dictionary(Of String, Object)

    End Class

End Namespace

File: Common/InMotionGIT.Common/Services/Contracts/StoredProcedureResult.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace InMotionGIT.Common.Services.Contracts
{

    [DataContract()]
    public class StoredProcedureResult
    {

        [DataMember()]
        public int RowAffected { get; set; }

        [DataMember()]
        public Dictionary<string, object> OutParameter { get; set; } = new Dictionary<string, object>();

    }

}